TanukiKit/Repositories.swift

```diff
@@ -5,60 +5,459 @@ import RequestKit
+//    }
+//}
+
 public extension TanukiKit {
     /**
-     Fetches the Repositories for the current user
-     - parameter owner: The user or organization that owns the repositories. If `nil`, fetches repositories for the authenticated user.
-     - parameter page: Current page for repository pagination. `1` by default.
-     - parameter perPage: Number of repositories per page. `100` by default.
+     Fetches the Projects for which the authenticated user is a member.
+     - parameter page: Current page for project pagination. `1` by default.
+     - parameter perPage: Number of projects per page. `100` by default.
+     - parameter archived: Limit by archived status. Default is `""`, set to `true` to enable.
+     - parameter visibility: Limit by visibility `public`, `internal` or `private`. Default is `""`
+     - parameter orderBy: Return projects ordered by `id`, `name`, `path`, `created_at`, `updated_at`, or `last_activity_at` fields. Default is `created_at`.
+     - parameter sort: Return projects sorted in asc or desc order. Default is `desc`.
+     - parameter search: Return list of authorized projects matching the search criteria. Default is `""`
+     - parameter simple: Return only the ID, URL, name, and path of each project. Default is `""`, set to `true` to enable.
      - parameter completion: Callback for the outcome of the fetch.
      */
-    public func repositories(_ session: RequestKitURLSession = URLSession.shared, page: String = "1", perPage: String = "100", completion: @escaping (_ response: Response<[Repository]>) -> Void) -> URLSessionDataTaskProtocol? {
-        let router = RepositoryRouter.readAuthenticatedRepositories(configuration, page, perPage)
+
+    // TODO: projects convertion.
+    public func repositories(_ session: RequestKitURLSession = URLSession.shared, page: String = "1", perPage: String = "20", archived: String = "", visibility: String = "", orderBy: String = "created_at", sort: String = "desc", search: String = "", simple: String = "", completion: @escaping (_ response: Response<[Repository]>) -> Void) -> URLSessionDataTaskProtocol? {
+        let router = ProjectRouter.readAuthenticatedProjects(configuration: configuration, page: page, perPage: perPage, archived: archived, visibility: visibility, orderBy: orderBy, sort: sort, search: search, simple: simple)
         return router.loadJSON(session, expectedResultType: [[String: AnyObject]].self) { json, error in
             if let error = error {
                 completion(Response.failure(error))
             }
 
             if let json = json {
+                // TODO: Change to projects.
                 let repos = json.map { Repository($0) }
                 completion(Response.success(repos))
             }
         }
     }
+
+    /**
+     Fetches project for a specified ID.
+     - parameter id: The ID or NAMESPACE/PROJECT_NAME of the project.
+     - parameter completion: Callback for the outcome of the fetch.
+     */
+
```

Comment:
Remove empty lines between documentation and method as Xcode does not pick it up otherwise.